release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -2504,3 +2504,36 @@ You can view the container images in this release by running the following comma
 $ oc adm release info 4.11.0 --pullspecs
 ----
 //replace 4.y.z for the correct values for the release. You do not need to update oc to run this command.
+
+[id="ocp-4-11-1"]
+=== RHSA-2022:6103 - {product-title} 4.11.1 bug fix and security update
+
+Issued: 2022-08-23
+
+{product-title} release 4.11.1, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2022:6103[RHSA-2022:6103] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2022:6102[RHSA-2022:6102] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory. See the following article for notes on the container images in this release:
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.11.1 --pullspecs
+----
+
+[id="ocp-4-11-1-bug-fixes"]
+==== Bug fixes
+
+* Previously, the functionality of Bond-CNI was limited to only active-backup mode. With this update, the bonding modes supported are:
+
+** balance-rr - 0
```

Comment:
Please double-check that these modes don't require [backticks](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference).

Reply:
oops yes it does! Thanks!